src/Router/Router.jsx

```diff
@@ -3,13 +3,15 @@ import Home from "src/pages/Home/Home";
 import SearchPage from "src/pages/Search/Search";
 import Categories from "src/pages/Categories/Categories";
 import Login from "src/pages/Login/Login";
+import Recipe from "src/pages/Recipe/Recipe";
 
 const Router = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/search" element={<SearchPage />} />
 		<Route path="/categories" element={<Categories />} />
 		<Route path="/login" element={<Login />} />
+      <Route path="/recipe" element={<Recipe />} />
```

Comment:
I know it's picky, but it would be great if you could format the file with prettier...!

Reply:
It is formatted on my VS Code, I dont know why it appears like that...
But if I see any code that is not formatted I will make them look cleaner and prettier.

Reply:
Oh I see, it's weird,,,